setup.py

```diff
@@ -65,8 +65,8 @@ def get_extras_require() -> Dict[str, List[str]]:
             "pytorch-ignite",
             "scikit-image",
             "scikit-learn",
-            "torch",
-            "torchvision>=0.5.0",
+            "torch==1.4.0+cpu",
+            "torchvision>=0.5.0+cpu",
```

Comment:
```
Collecting torchvision>=0.5.0+cpu
507
  Downloading https://download.pytorch.org/whl/cu92/torchvision-0.5.0%2Bcu92-cp37-cp37m-linux_x86_64.whl (3.9 MB)
```

https://github.com/hvy/optuna/pull/5/checks?sha=306b2e89defadc4685ae860ae64c05e99b4a0ead

It could be better to use `==` for `torchvision` as well.
(Although I'm not sure whether it matters, `torchvision>=0.5.0+cpu` install torchvision with CUDA)


```
$ pip install 'torchvision==0.5.0+cpu' -f https://download.pytorch.org/whl/torch_stable.html
Looking in links: https://download.pytorch.org/whl/torch_stable.html
Collecting torchvision==0.5.0+cpu
```